<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogAltaPersonal</class>
 <widget class="QDialog" name="DialogAltaPersonal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Alta Personal</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Nuevo Empleado</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="labelAlignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Nombre:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Apellido:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEditNombre"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Nro Documento:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEditApellido"/>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="lineEditNroDocumento"/>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="lineEditDomicilio"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Domicilio:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Tipo de Documento:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="comboBoxTipoDocumento"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Fecha de Nacimiento:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QDateEdit" name="dateEditFechaNacimiento"/>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Telefono:</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Email:</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLineEdit" name="lineEditTelefono"/>
      </item>
      <item row="7" column="1">
       <widget class="QLineEdit" name="lineEditEmail"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxButtonBox">
     <property name="minimumSize">
      <size>
       <width>271</width>
       <height>41</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>41</height>
      </size>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>9</number>
      </property>
      <property name="topMargin">
       <number>9</number>
      </property>
      <property name="rightMargin">
       <number>9</number>
      </property>
      <property name="bottomMargin">
       <number>8</number>
      </property>
      <item row="0" column="1">
       <widget class="QPushButton" name="pushButtonAceptar">
        <property name="text">
         <string>Aceptar</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="pushButtonCancelar">
        <property name="text">
         <string>Cancelar</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEditNombre</tabstop>
  <tabstop>lineEditApellido</tabstop>
  <tabstop>comboBoxTipoDocumento</tabstop>
  <tabstop>lineEditNroDocumento</tabstop>
  <tabstop>dateEditFechaNacimiento</tabstop>
  <tabstop>lineEditDomicilio</tabstop>
  <tabstop>lineEditTelefono</tabstop>
  <tabstop>lineEditEmail</tabstop>
  <tabstop>pushButtonAceptar</tabstop>
  <tabstop>pushButtonCancelar</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
